settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':app'
+include ':familyplanningmodule'
```

Comment:
:thinking: Do you think that is would be better to name the family planning as `opensrp-fp` instead of `familyplanningmodule`?
This follows the OpenSRP convention of naming projects & modules

Make this change to the directory name of the module also